pyrealm/pmodel/two_leaf_irradience.py

```diff
@@ -0,0 +1,31 @@
+"""To do."""
+
+import numpy as np
+from numpy.typing import NDArray
+
+
+class TwoLeafIrradience:
+    """Calculates two-leaf irradience."""
+
+    def __init__(self, beta_angle: NDArray, PPFD: NDArray, LAI: NDArray, PATM: NDArray):
```

Comment:
Although these variable names are acronyms, I think we should stick to lower case variable names: `ppfd`, `lai`, `patm` (and arguably longer - `leaf_area_index`, although I know I'm guilty of using `patm` and `tc` elsewhere. 

We could do with a sweep to standardise /check variable names, really!